### ISSUE TITLE
package-build-package is now private

### ISSUE THREAD

**Issue by GuiltyDolphin** [cask/cask#426]
A [recent change](https://github.com/melpa/package-build/commit/7733c252691c2c1b8491c375176ca51a6e0dfd33#diff-ac4dc19e4f6618a58fcf1c0980e31857R751) to `package-build` renamed `package-build-package` to `package-build--package` (presumably to make it 'private').

This fails when specifying a fetcher, e.g., with the `Cask` file:

```emacs-lisp
(depends-on
  "cask"
  :git "git://github.com/cask/cask.git")
```

Running `cask install` gives:

```
Dependency cask failed to install: Symbol’s function definition is void: package-build-package
Output:

```

I'm not entirely sure what is intended to be used in place of this, but presumably `package-build-package` shouldn't be used any more.

**Comment by rcoedo** [comment]
I'm having the same problem. Any workaround or suggestion?

**Comment by GuiltyDolphin** [comment]
@rcoedo As a temporary work around, simply replacing references to `package-build-package` with `package-build--package` works. Looking at some of the other issues/pull requests that reference this, I think there may be a more permanent solution in the future.

**Comment by sambrightman** [comment]
Fixed by #429 for now.
